File: py/updown_game.py
import random

print("Up/Down Game v1.0")
print("Copyright (c) 2020 miniprime1\n")

print("Difficult")
print("1. Easy (1~10)")
print("2. Normal (1~100)")
print("3. Hard (1~1000)")
print("4. Extreme (1~10000)")
print("5. Custom (?~?)")
opt = input("Your choice: ")

if int(opt) == 1: 
    ans = random.randrange(1, 11)
    print("\nDifficult: Easy (1~10)")
elif int(opt) == 2: 
    ans = random.randrange(1, 101)
    print("\nDifficult: Normal (1~100)")
elif int(opt) == 3: 
    ans = random.randrange(1, 1001)
    print("\nDifficult: Hard (1~1000)")
elif int(opt) == 4: 
    ans = random.randrange(1, 10001)
    print("\nDifficult: Extreme (1~10000)")
elif int(opt) == 5:
    x = input("\nEnter 1st input: ")
    y = input("Enter 2nd input: ")
    print("\nDifficult: Custom (" + x + "~" + y + ")")
    ans = random.randrange(int(x), int(y) + 1)
else: 
    ans = random.randrange(1, 1000001)
    print("\nDifficult: Error (1~1000000)")

while True:
    you = int(input("\nEnter number: "))
    if you > ans: print("Down!")
    if you < ans: print("Up!")
    if you == ans: break

print("Answer!")

input("\nPress enter key to exit")

File: java/UpdownGame.java
import java.util.*;
import static java.lang.System.*;

public class UpdownGame {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        Random rnd = new Random();
        int opt, ans, x, y;
        out.println("Up/Down Game v1.0");
        out.println("Copyright (c) 2020 miniprime1\n");
        out.println("Difficult");
        out.println("1. Easy (1~10)");
        out.println("2. Normal (1~100)");
        out.println("3. Hard (1~1000)");
        out.println("4. Extreme (1~10000)");
        out.println("5. Custom (?~?)");
        out.print("Your choice: ");
        opt = Integer.parseInt(input.nextLine());

        if (opt == 1) {
            out.println("\nDifficult: Easy (1~10)");
            ans = rnd.nextInt(10) + 1;
        } else if (opt == 2) {
            out.println("\nDifficult: Normal (1~100)");
            ans = rnd.nextInt(100) + 1;
        } else if (opt == 3) {
            out.println("\nDifficult: Hard (1~1000)");
            ans = rnd.nextInt(1000) + 1;
        } else if (opt == 4) {
            out.println("\nDifficult: Extreme (1~10000)");
            ans = rnd.nextInt(10000) + 1;
        } else if (opt == 5) {
            out.print("\nEnter 1st input: ");
            x = Integer.parseInt(input.nextLine());
            out.print("Enter 1st input: ");
            y = Integer.parseInt(input.nextLine());
            out.println("\nDifficult: Custom (" + x + '~' + y + ")");
            ans = rnd.nextInt(10) + 1;
        } else {
            out.println("\nError: invalid choice");
            out.println("Difficult will be a Normal (1~100)");
            ans = rnd.nextInt(10) + 1;
        }

        while (true) {
            out.print("\nEnter number: ");
            int user = Integer.parseInt(input.nextLine());
            if (user > ans) {out.println("Down!");}
            if (user < ans) {out.println("Up!");}
            if (user == ans) { out.println("Answer!"); break; }
        }

        input.close();
    }
}